app/views/merchant/orders/show.html.erb

```diff
@@ -6,18 +6,18 @@
     <%= @order.city %><br>
     <%= @order.state %><br>
     <%= @order.zip %><br>
-    <% @order.items.each do |item| %>
-      <section id= <%= "item-#{item.id}" %>>
-      <% if item.merchant_id == self.current_user.merchant_id %>
-        <%= item.name %><br>
-        <img src=<%= item.image %>><br>
-        <%= item.price %><br>
-        <%= "Number of Items: #{item.item_orders[0].quantity}" %><br>
-        <%= item.item_orders[0].status.capitalize %><br>
-        <% if item.item_orders[0].quantity <= item.inventory && item.item_orders[0].unfulfilled? %>
-          <%= button_to "Fulfill", "/item_orders/#{item.item_orders[0].id}", params: {fulfill: "fulfill"}, method: :patch %>
+    <% @order.item_orders.each do |item_order| %>
+      <% if item_order.item.merchant_id == self.current_user.merchant_id %>
+      <section id= <%= "item-#{item_order.item.id}" %>>
+        <%= item_order.item.name %><br>
+        <img src=<%= item_order.item.image %>><br>
+        <%= item_order.item.price %><br>
+        <%= "Number of Items: #{item_order.quantity}" %><br>
+        <%= item_order.status.capitalize %><br>
+        <% if item_order.unfulfilled? && item_order.quantity <= item_order.item.inventory%>
```

Comment:
we should have had a helper method here instead the logic in the view. We need to refactor this